src/app.ts

```diff
@@ -0,0 +1,61 @@
+import express, { Request, Response } from "express";
+import nunjucks from "nunjucks";
+import path from "path";
+import logger from "./lib/Logger";
+import routerDispatch from "./router.dispatch";
+
+const app = express();
+
+const viewPath = path.join(__dirname, "/views");
+app.set("views", viewPath);
+
+const nunjucksLoaderOpts = {
+    watch: process.env.NUNJUCKS_LOADER_WATCH !== "false",
+    noCache: process.env.NUNJUCKS_LOADER_NO_CACHE !== "true"
+};
+
+const njk = new nunjucks.Environment(
+    new nunjucks.FileSystemLoader(app.get("views"),
+        nunjucksLoaderOpts)
+);
+
+njk.express(app);
+app.set("view engine", "njk");
+
+// Serve static files
+app.use(express.static(path.join(__dirname, "/../assets/public")));
+// app.use("/assets", express.static("./../node_modules/govuk-frontend/govuk/assets"));
```

Comment:
Wondering why this line is commented out? We should just delete it, or, if it remains (because it's useful for dev purposes?), it needs a comment above it saying why it's retained.

Reply:
This was copied over from the template. I'll remove the line.